skbeam/core/accumulators/binned_statistic.py

```diff
@@ -0,0 +1,445 @@
+from __future__ import division, print_function, absolute_import
+
+import warnings
+
+import numpy as np
+from scipy._lib.six import callable
+from collections import namedtuple
+
+
+BinnedStatisticResult = namedtuple('BinnedStatisticResult',
+                                   ('statistic', 'bin_edges', 'binnumber'))
+
+BinnedStatisticddResult = namedtuple('BinnedStatisticddResult',
+                                     ('statistic', 'bin_edges',
+                                      'binnumber'))
+
+
+class BinnedStatistic(object):
+    def __init__(self, x, statistic='mean',
+                 bins=10, range=None):
+        """
+        Compute a binned statistic for a set of data.
+
+        This is a generalization of a histogram function.  A histogram divides
+        the space into bins, and returns the count of the number of points in
+        each bin.  This function allows the computation of the sum, mean,
+        median, or other statistic of the values within each bin.
+
+        Parameters
+        ----------
+        values : array_like
```

Comment:
You are right.  I hadn't re-thought the scipy documentation well enough after re-factoring.  I have tried to do that in the most recent commits.